app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -142,7 +142,15 @@ class NewProfileFragment : Fragment() {
                 while (i < 7) {
                     if (days[i]) {
                         sAlarm(i + 1, profile)
-                        eAlarm(i + 1, profile)
+                        if ((shr > ehr) || ((shr == ehr) && (smin > emin))) {
```

Comment:
@mouryac9 I want to ask one thing. Why include the  ((shr == ehr) && (smin > emin) in the if condition?
The rest looks good to me and I tested it today and it worked as expected.

Reply:
Initially it was there to check that starting time is not greater than ending time. But now I guess it won't be of much use now that there's a 12 hour restriction. I'll remove it.